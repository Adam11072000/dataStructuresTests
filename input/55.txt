init 8915
quit