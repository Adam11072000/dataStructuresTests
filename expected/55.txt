init done.
deleteImage: FAILURE
setLabelScore: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
quit done.
